force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +311,16 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
-
+        Integer wordIndex = -1;
         // Loop through the list of words
             // If the current word is the word to find, exit the loop
-
-        return null; // Replace null with the variable you used to store the result
+         for (String word : words) {
+             if(word == wordToFind){
+                wordIndex = words.indexOf(word);
+                break;
```

Comment:
Index of is great but for practice try iterating over the values and finding the word in the list